### DISCUSSION TITLE
Does portman supports form-data ?

### DISCUSSION THREAD

**Discussion by yajiv789** [apideck-libraries/portman#245] [Q&A] [answered]
I am trying to create contract tests for an endpoint which has content-Type as form-data. I defined configuration similar to application/json but it doesn't overwrite the request body. 

If portman supports please share the documentation. Thanks in advance.

**Comment by thim81** [comment, marked as answer]
@yajiv789 
Currently Portman, only support the Content-Type `application/json`.

Wa are actively reviewing what the workload would be and if it is feasible/desireable on extending the support for multiple content-types like `form-data`.

**Comment by thim81** [comment]
@yajiv789 Do you have a simple OpenAPI spec which contains the `form-data` and would you mind sharing it?

**Reply by yajiv789** [reply]
sure will share it soon.

**Comment by yajiv789** [comment]
@thim81  plz find attached sample form-data openapi yaml. 
[openapi-form-data.yaml.zip](https://github.com/apideck-libraries/portman/files/7887846/openapi-form-data.yaml.zip)

**Reply by yajiv789** [reply]
@thim81 Let me know if you need any more info.  Most of my endpoints has form-data. Much appreciated if this feature is added asap.

**Reply by thim81** [reply]
@yajiv789 Just for awareness, I havent found the time to look into the form-data feature and it will take some waiting since my schedule does not have plenty of room at this moment.

**Reply by yajiv789** [reply]
@thim81  sure, thanks for letting me know. Please notify me whenever you have any tentative plan to work on this feature.
